index.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>My Portfolio</title>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <header>
+            <img class="logo" alt="Logo" src="./images/image_geometry_1.png">
+            <img class="menu" alt="Menu" src="./images/ic_menu.svg">
+        </header>
+        <div class="headline">
+            <div class="name">Emily Robertson</div>
+            <div class="intro">Hello! I am a software developer! 
+                I can help you build a product, 
+                feature or website. Take a look at my works.
+                If you like what you see and have a project 
+                you need coded, don’t hesitate and contact me.
+            </div>
+            <button type="button">Start Collaboration</button>
+            <img class="banner" alt="Image" src="./images/image_geometry_2.png">
+        </div>
```

Comment:
- [ ] Great job implementing the markup, however, it is best to follow [HTML & CSS best practices,](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) one of which is to use HTML5 semantic tags (e.g., header, nav) over `div`s.